src/devices/src/virtio/vsock/csm/connection.rs

```diff
@@ -749,6 +749,17 @@ mod tests {
         }
     }
 
+    #[cfg(test)]
```

Comment:
You don't need to specify the `cfg(test)` macro because this implementation is defined in `mod tests` which is all only compiled for tests.

Reply:
Removed